server/apps/immich/src/config/asset-upload.config.ts

```diff
@@ -52,16 +50,20 @@ function fileFilter(req: Request, file: any, cb: any) {
   if (!req.user || (req.user.isPublicUser && !req.user.isAllowUpload)) {
     return cb(new UnauthorizedException());
   }
-  if (
-    file.mimetype.match(
-      /\/(jpg|jpeg|png|gif|mp4|webm|x-msvideo|quicktime|heic|heif|dng|x-adobe-dng|webp|tiff|3gpp|nef|x-nikon-nef)$/,
-    )
-  ) {
-    cb(null, true);
-  } else {
-    logger.error(`Unsupported file type ${extname(file.originalname)} file MIME type ${file.mimetype}`);
-    cb(new BadRequestException(`Unsupported file type ${extname(file.originalname)}`), false);
-  }
+  // TODO: Create new API endpoint for mimetypes and use that here as browser's
```

Comment:
Can we uncommented this?

Reply:
Unfortunately with it uncommented, those files will fail to be uploaded due to the mimetype issue from browser.

Reply:
Uncommented and added handling for the srw and raf formats even if browser reports them incorrectly.